libs/hdf-converters/src/xccdf-results-mapper.ts

```diff
@@ -213,9 +218,7 @@ export class XCCDFResultsMapper extends BaseConverter {
       }
     },
     version: HeimdallToolsVersion,
-    statistics: {
-      duration: 0
-    },
+    statistics: {},
     profiles: [
       {
         name: {path: ['cdf:Benchmark.id', 'Benchmark.id']},
```

Comment:
@camdenmoors mentioned something about how `cdf:` is some sorta prefix that we can tell the parser to ignore.  talk to him to see what's up with that so we can ideally simplify this mapper.